AllInOne.cs

```diff
@@ -176,22 +178,20 @@ public A(int foo)
             var o1 = new MyObject();
             var o2 = new MyObject(var);
             var o3 = new MyObject { A = i };
-            var o4 = new MyObject(@dynamic) { A = 0, B = 0, C = 0 };
-            var o5 = new { A = 0 };
-            var dictionaryInitializer = new Dictionary<int, string>
+            var o4 = new MyObject(@dynamic)
             {
-                { 1, "" },
-                { 2, "a" }
+                A = 0,
+                B = 0,
+                C = 0
             };
+            var o5 = new { A = 0 };
+            var dictionaryInitializer = new Dictionary<int, string> { { 1, "" }, { 2, "a" } };
             float[] a = new float[] { 0f, 1.1f };
-            int[,,] cube = {
-                { { 111, 112 }, { 121, 122 } },
-                { { 211, 212 }, { 221, 222 } }
-            };
+            int[,,] cube = { { { 111, 112 }, { 121, 122 } }, { { 211, 212 }, { 221, 222 } } };
```

Comment:
Hmm... is this intended to force us to encapsulate object construction? Is there any way to have dictionary initialization span multiple lines?